src/main/java/uk/nhs/digital/nhsconnect/nhais/utils/Headers.java

```diff
@@ -0,0 +1,5 @@
+package uk.nhs.digital.nhsconnect.nhais.utils;
+
+public interface Headers {
```

Comment:
I'd add context to this name (like `JmsHeaders`)- we might have many different headers :)

Reply:
And why interface? Class/enum would work - no one should "implement" `Headers`

Reply:
So if the same header is used in both http and jms, there should be 2 classes with the same value?

Reply:
Yeah, why not? They're completely different things, besides the name. What if one had to be renamed?